lib/rules/value-keyword-case/__tests__/index.js

```diff
@@ -1938,3 +1938,79 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
```

Comment:
Please include a couple of regular expression tests, like [we do](https://github.com/Bilie/stylelint/blob/140f09771b9eb26538537e75469a850313ea617b/lib/rules/value-keyword-case/__tests__/index.js#L1877-L1940) for the `ignoreKeywords` option.